app/book.py

```diff
@@ -0,0 +1,14 @@
+class Book:
+    def __init__(self, title: str, content: str):
+        self.title = title
+        self.content = content
+
+    def print_book(self, print_type: str) -> None:
+        if print_type == "console":
+            print(f"Printing the book: {self.title}...")
+            print(self.content)
+        elif print_type == "reverse":
+            print(f"Printing the book in reverse: {self.title}...")
+            print(self.content[::-1])
+        else:
+            raise ValueError(f"Unknown print type: {print_type}")
```

Comment:
The `print_book` method currently handles only 'console' and 'reverse' as valid `print_type` values. Ensure that these are the only expected values according to the task requirements. If there are additional print types expected, you should handle them accordingly. Otherwise, the implementation is correct for the given functionality.